### PR TITLE
Add recipe for OOP Extensions

### PR DESCRIPTION
**Pull request by lincrosenbach**

### CODE REVIEW

File: recipes/oop-ext/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  license_family: MIT
+  license_file: LICENSE
+  summary: OOP Extensions is a set of utilities for object oriented programming which is missing on Python core libraries.
+  doc_url: https://oop-ext.readthedocs.io/
+  dev_url: https://github.com/ESSS/oop-ext
+
+extra:
+  recipe-maintainers:
+    - lincrosenbach
+    - kfasolin
```
Comment (kfasolin) [inline recipes/oop-ext/meta.yaml]: I'm ok being a maintainer

File: recipes/oop-ext/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - nicoddemus
```
Comment (nicoddemus) [inline recipes/oop-ext/meta.yaml]: I'm OK being maintainer here. 👍

**Comment by nicoddemus** [comment]
@ocefpaf @conda-forge/staged-recipes I believe this is ready for review/merge (it is a noarch package so there's no need to wait for Travis). 👍

**Comment by marcelotrevisani** [comment]
LGTM!
Thanks for adding this! 🎉 🎉 🎉

File: recipes/oop-ext/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dd5982b0d677b585c76996cae5470c469325f9275ae3317fceabdfef2c3329ec
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (ocefpaf) [inline recipes/oop-ext/meta.yaml]: We need to `--no-deps` there. See https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip

**Pull request merged**